### PR TITLE
actions: Create release on push

### PR DESCRIPTION
**Pull request by dracc**

### CODE REVIEW

**Comment by MasonT8198** [comment]
Looks good to me @mborgerson

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -84,3 +101,95 @@ jobs:
...
+        path: dist
+    - name: Create archives
+      run: |
+        pushd dist/extract-xiso_Win32_Release
+        zip -r ../extract-xiso-win32-release.zip *
+        popd
+        pushd dist/extract-xiso_Win32_Debug
+        zip -r ../extract-xiso-win32-debug.zip *
+        popd
+        pushd dist/extract-xiso_Win64_Release
```
Comment (GXTX) [inline .github/workflows/CI.yml:124]: Should the win64 builds be commented out for now as #29 still applies?

Comment (mborgerson) [reply .github/workflows/CI.yml:124]: I agree, let's comment this out and not distribute broken 64b builds for now

Comment (dracc) [reply .github/workflows/CI.yml]: Consider it done.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -84,3 +101,101 @@ jobs:
...
+    #   id: upload-release-assets-win64-debug
+    #   uses: actions/upload-release-asset@v1.0.1
+    #   env:
+    #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    #   with:
+    #     upload_url: ${{ steps.create_release.outputs.upload_url }}
+    #     asset_name: extract-xiso-win64-debug.zip
+    #     asset_path: dist/extract-xiso-win64-debug.zip
+    #     asset_content_type: application/zip
+    - name: Upload release assets (macOS build)
```
Comment (mborgerson) [inline .github/workflows/CI.yml:192]: Let's also not distribute these 64b Mac builds until 64b is fixed

Comment (dracc) [reply .github/workflows/CI.yml]: Done, updated comments accordingly.

**Comment by mborgerson** [comment]
Thanks!

**Pull request merged**